scripts/data_store.py

```diff
@@ -0,0 +1,59 @@
+import os
+import shelve
+import memory
+import message_history
+
+class DataStore():
+
+    def persist_message_history(self, id):
+      return True
```

Comment:
What about return False on everything for the abstract base class? Or if errors are involved, throwing the NotImplementedError?